sql-statements/sql-statement-restore.md

```diff
@@ -103,17 +103,7 @@ BR supports restoring data from S3 or GCS:
 RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/';
 ```
 
-<CustomContent platform="tidb">
-
-The URL syntax is further explained in [external storage URI](/br/backup-and-restore-storages.md#uri-format).
-
-</CustomContent>
-
-<CustomContent platform="tidb-cloud">
-
-The URL syntax is further explained in [external storage URI](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages#uri-format).
-
-</CustomContent>
+The URI syntax is further explained in [URI Formats of External Storage Services](/external-storage-uri.md).
```

Comment:
As `sql-statements/sql-statement-import-into.md` is also used by TiDB Cloud TOC, please use the CustomContent tag for this paragraph.